### PR TITLE
Instance test enhancments

### PR DESCRIPTION
**Pull request by ShahafBenYakir**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Description
Some small fixes to instance test step including the creation of an artifact that includes all failing tests in addition to the list.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -560,6 +560,7 @@ jobs:
             if [ -n "${INSTANCE_TESTS}" ] && [ $CIRCLE_NODE_INDEX -ne 1 ] ;
               then
                 ./Tests/scripts/instance_test.sh
+                cp ./Tests/failed_instances.txt $CIRCLE_ARTIFACTS/failed_instances.txt
```
Comment (yaakovi) [inline .circleci/config.yml:563]: this is hiding the error code of the bash script, need to handle it like we do [here](https://github.com/demisto/content/pull/7624/files#diff-1d37e48f9ceff6d8030570cd36286a61R530-R532)

Comment (ShahafBenYakir) [reply .circleci/config.yml:563]: Is this an escape room? link leads to nowhere...

Comment (yaakovi) [reply .circleci/config.yml:563]: see lines 530-532 in this file.

**Comment by ShahafBenYakir** [comment]
Failing on MP step. Force merging.
FYI @yuvalbenshalom

**Pull request merged**